apps/web/next.config.js

```diff
@@ -52,6 +52,15 @@ const withTMExports = withTM(
       NEXT_PUBLIC_NODE_ENV: process.env['NODE_ENV'],
     },
     reactStrictMode: true,
+    async redirects() {
+      return [
+        {
+          source: '/',
+          destination: '/pipelines',
```

Comment:
I'm not sure if this is appropriate, what if we wanted to redirect you to the `auth` page for example?